pandas/io/formats/format.py

```diff
@@ -1176,6 +1176,11 @@ def _format_strings(self):
                                   na_rep=self.na_rep, digits=self.digits,
                                   space=self.space, justify=self.justify,
                                   leading_space=self.leading_space)
+
```

Comment:
we have a CategoricalFormatter already, mod should be made there. Should be caught when formatting, not as a post-action.